### PR TITLE
Ticket Fixed Mapstruct

### PR DESCRIPTION
**Pull request by walterio123**
**Summary**
- fixed Mapstruct Implement

### CODE REVIEW

File: src/main/java/com/alkemy/ong/controller/CategoryController.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/category")
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/controller/CategoryController.java:15]: Also I would rather call /categories strait

File: src/main/java/com/alkemy/ong/entity/CategoryEntity.java [hunk 2]
```diff
@@ -11,7 +11,6 @@
 @Entity
 @Getter @Setter @ToString @RequiredArgsConstructor
 @SQLDelete(sql="UPDATE categorys SET softDelete = true WHERE id=?")
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/entity/CategoryEntity.java:13]: I suggest that the table name changes for categories in stead of categorys (just a spelling thing) but we have to consider dropping our DB

Comment (Lucasfranco1) [reply src/main/java/com/alkemy/ong/entity/CategoryEntity.java:13]: I think the name complies with what is stated on the ticket. It's fine with me to leave it like that.

**Comment by Lucasfranco1** [comment]
Plase, resolve the conflict.

**Pull request merged**